recipes/syngap/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "syngap" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/yanyew/bioconda/releases/download/v{{ version }}/syngap_v{{ version }}.tar.gz
+  sha256: a2848c925d8df49872ba23f4e0afbc176ff1688b60b04ee3c5f318d41dcc6c69
+  
+build: 
+  number: 0
+  entry_points: 
+    - syngap = SynGAP:main
+  noarch: generic
+  run_exports:
+    - python
+    - biopython
+    - jcvi
+    - bedtools
+    - last
+    - emboss
+    - gffread
+    - seqkit
+    - diamond
+    - perl-bioperl
+    - kneed
+    - numpy
+    - pandas
+    - matplotlib-base
+
+requirements:
+  bulid: 
+    - python >=3.10
+  run: 
+    - python >=3.10
+    - biopython >=1.81
+    - jcvi >=1.3.6
+    - bedtools >=2.31.0
+    - last >=1454
+    - emboss >=6.6.0
+    - gffread >=0.12.7
+    - seqkit >=2.4.0
+    - diamond >=2.1.8
+    - perl-bioperl >=1.7.8
+    - kneed >=0.8.3
+    - numpy >=1.26.0
+    - pandas >=2.1.1
+    - matplotlib-base >=3.8.0
+    - scikit-image >=0.22.0
+    - pybedtools >=0.9.0
+    - deap >=1.4.1
+  pip:
+    - more-itertools
+    - ftpretty
+    - ortools
+    - graphviz
+    - webcolors
+    - CrossMap
+
+test: 
+  source_files: 
+    - run_test.sh
+    - SynGAP.py
+    - dual.py
+    - triple.py
+    - custom.py
+    - genepair.py
+    - evi.py
+    - eviplot.py
+    - bin/*
+    - scripts/*
+  imports: 
+    - SynGAP
+  commands: 
+    - syngap -h
+    - syngap dual -h
+    - syngap triple -h
+    - syngap custom -h
+    - syngap genepair -h
+    - syngap evi -h
+    - syngap eviplot -h
+
+about: 
+  home: https://github.com/yanyew/SynGAP
```

Comment:
Sorry for this case. Because the manuscript of this software is still not published, I set the github repository Private.
Do you need to check the source code?

Reply:
Thanks but no thanks, I can see the code from this link `https://github.com/yanyew/syngap_bioconda/releases/download/v{{ version }}/syngap_v{{ version }}.tar.gz` , can't I? So I don't see why you want to keep it private, you also license this software as CC-BY-NC-SA-4.0
I see in the compressed file a swiss prot db with ~ 290MB, is it a special build or can it be downloaded from a public source once the software is installed?
@bgruening any thoughts?

Reply:
I have changed the repository into Public.
As for the prepared swiss-prot db, it isn't a special build. I pack it just for convenient use. Should I remove it?

Reply:
If possible, I'd keep the database somewhere and show a message from the software to end users that they will need to download that database.